src/services/projects.js

```diff
@@ -470,131 +523,334 @@ module.exports = class ProjectsHelper {
 				['id', 'value', 'has_entities', 'validations']
 			)
 
-			// Using forEach instead of for loop for entity type validations
+			//fetch task entityType validations
+			let taskEntityTypes = await entityModelMappingQuery.findEntityTypesAndEntities(
+				{
+					model: common.TASKS,
+					status: common.STATUS_ACTIVE,
+				},
+				userDetails.organization_id,
+				['id', 'value', 'validations', 'has_entities']
+			)
+
+			const taskEntityTypesMapping = taskEntityTypes.reduce((acc, item) => {
+				acc[item.value] = item
+				return acc
+			}, {})
+
+			//get all entity type validation for file types
+			let allowedFileTypesEntityTypes = await entityTypeQueries.findUserEntityTypeAndEntities(
```

Comment:
Why is the same query done here and on line 527, file validations also should have models

Reply:
i will change it